components/v1/HomePage/HomePageMeta.tsx

```diff
@@ -0,0 +1,35 @@
+import type { FunctionComponent } from "react";
+
+/**
+ * HomePageMetaProps is a React Component properties that passed to React Component HomePageMeta
+ */
+ type HomePageMetaProps = {};
+
+ /**
+  * HomePageMeta is just yet another react component
+  *
+  * @link https://fettblog.eu/typescript-react/components/#functional-components
+  */
+
+ const HomePageMeta: FunctionComponent<HomePageMetaProps> = ({}) => {
+    return(
+        <>
+            {/* <!-- Facebook Meta Tags --> */}
+            <meta property="og:url" content="https://risedle.com/"/>
+            <meta property="og:type" content="website"/>
+            <meta property="og:title" content="Risedle Protocol"/>
+            <meta property="og:description" content="Invest, earn and build on the decentralized crypto leveraged ETFs market protocol"/>
+            <meta property="og:image" content="https://risedle.com/Landing.png"/>
```

Comment:
@math-marcellino mathew, ini gambar OG nya bisa di jadiin satu folder semuanya ga?

```
public/og/homepage.png
public/og/markets.png
public/og/ethrise.png
```

Reply:
Okesiap mas bentar ku fix ya, thank you for the review! @pyk